widgetssdk/src/main/java/com/glia/widgets/helper/ActivityWatcherForDialogs.kt

```diff
@@ -15,9 +15,9 @@ import com.glia.widgets.view.Dialogs
 import com.google.android.material.theme.overlay.MaterialThemeOverlay
 import java.lang.ref.WeakReference
 
-class ActivityWatcherForDialogs(
+internal class ActivityWatcherForDialogs(
```

Comment:
Agree with you, I realized this after I finished the changes. It will be complicated to merge changes because of changes on both the `call visualizer` and `secure conversations` sides. But however, I need some of them because the current version is causing issues related to `DialogController` work on the Message Center side.